common/test/acceptance/tests/lms/test_lms_split_test_courseware_search.py

```diff
@@ -161,8 +123,5 @@ def test_search_for_experiment_content_user_assigned_to_one_group(self):
         """
         self._auto_auth(self.USERNAME, self.EMAIL, False)
         self.courseware_search_page.visit()
-        self.course_navigation_page.go_to_section("Test Section", "Test Subsection")
-        self.courseware_search_page.search_for_term("Group")
+        self.courseware_search_page.search_for_term("VISIBLE TO")
         assert "1 result" in self.courseware_search_page.search_results.html[0]
-        assert self.test_1_breadcrumb in self.courseware_search_page.search_results.html[0]
-        assert self.test_2_breadcrumb not in self.courseware_search_page.search_results.html[0]
```

Comment:
The breadcrumb test seemed fragile, what if change how breadcrumbs are displayed? I also removed the "X Section" block and the second partition, which I feel doesn't make sense to test now that we are using the mobile api.